lib/XTR/ReportScheduleTask.php

```diff
@@ -214,7 +221,7 @@ private function runReportSchedule()
      * @throws \Twig\Error\SyntaxError
      * @throws \Xibo\Support\Exception\GeneralException
      */
-    private function createPdfAndNotification($reportSchedule, $savedReport, $media)
+    private function createPdfAndNotification($reportSchedule, $savedReport, $runDateTimestamp)
```

Comment:
I wonder if we perhaps should use `$savedReport` id here instead of the timestamp? Perhaps with some prefix if needed

Reply:
I agree.